Libplanet/Net/Swarm.cs

```diff
@@ -81,6 +78,7 @@ static Swarm()
         /// <param name="trustedAppProtocolVersionSigners"><see cref="PublicKey"/>s of parties
         /// to trust <see cref="AppProtocolVersion"/>s they signed.  To trust any party, pass
         /// <c>null</c>, which is default.</param>
+        /// <param name="swarmOptions">Options for <see cref="Swarm{T}"/>.</param>
```

Comment:
As it's of course swarm's options, how about simply renaming it to `options`?

Reply:
Changed it.